# HG changeset patch
# User Tartalo
# Date 1278534638 -7200
# Branch trunk
# Node ID cb780b69adb2f2c2f0b7a7cf9c8a686a550ad2b3
# Parent  8590d3da443039a427436a8e115ac3b2875f40ae
Coment annoying assert that nobody knows why is there

diff -r 8590d3da4430 -r cb780b69adb2 src/server/game/Maps/Map.cpp
--- a/src/server/game/Maps/Map.cpp	Wed Jul 07 22:22:16 2010 +0200
+++ b/src/server/game/Maps/Map.cpp	Wed Jul 07 22:30:38 2010 +0200
@@ -2369,7 +2369,7 @@
     if (player->GetMapRef().getTarget() == this)
     {
         sLog.outError("InstanceMap::CanEnter - player %s(%u) already in map %d,%d,%d!", player->GetName(), player->GetGUIDLow(), GetId(), GetInstanceId(), GetSpawnMode());
-        assert(false);
+        //assert(false);
         return false;
     }
 
